app/src/main/java/org/fossasia/openevent/app/event/EventContainerFragment.java

```diff
@@ -74,6 +74,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         // Inflate the layout for this fragment
         View view = inflater.inflate(R.layout.fragment_event_container, container, false);
         ButterKnife.bind(this, view);
+
+        setRetainInstance(true);
```

Comment:
what is actual purpose of this?
If it is required, it should be called in `onCreate` instead.